themes-default/slim/static/js/core.js

```diff
@@ -36,7 +36,7 @@ const UTIL = {
         $('[asset]').each(function() {
             const asset = $(this).attr('asset');
             const series = $(this).attr('series');
-            const path = apiRoot + 'series/' + series + '/asset/' + asset + '?api_key=' + apiKey;
+            const path = apiRoot + `series/${series}/asset/${asset}?api_key=${apiKey}&${MEDUSA.config.pid}${MEDUSA.config.inc}`;
```

Comment:
This will essentially place the PID+INC value after every image request.
If for example the PID = 1234, we start with the INC value 1. The value will be 12341.
After refreshing an image, the incremental value will be bumped to 2. And the value 12342 is added, making it essentially refresh the image.